source/operations/checklists/thresholds.rst

```diff
@@ -0,0 +1,144 @@
+.. _minio-server-limits:
+
+=====================
+Thresholds and Limits
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+This page reflects limits and thresholds that apply to MinIO.
+
+Refer to the :ref:`hardware <minio-hardware-checklist>` and :ref:`software <minio-software-checklists>` for related recommendations and requirements.
+
+S3 API Limits
+-------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 75 25
+   :width: 90%
+
+   * - Item
+     - Specification 
+
+   * - Maximum number of buckets
+     - 500,000
+
+   * - Maximum number of objects per bucket
+     - no limit
+
+   * - Maximum object size
+     - 50 TiB
+
+   * - Minimum object size
+     - 0 B
+
+   * - Maximum object size per PUT operation
+     - | 5 TiB for non-multipart upload
+       | 50 TiB for multipart upload
+
+   * - Maximum number of parts per upload
+     - 10,000
+
+   * - Part size range
+     - 5 MiB to 5 GiB. Last part can be 0 B to 5 GiB
+
+   * - Maximum number of parts returned per list parts request
+     - 10,000
+
+   * - Maximum number of objects returned per list objects request
+     - 1,000
+
+   * - Maximum number of multipart uploads returned per list multipart uploads request
+     - 1,000
+
+   * - Maximum length for bucket names
+     - 63
+
+   * - Maximum length for object names
+     - 1024
+
+   * - Maximum length for each ``/`` separated object name segment
+     - 255
+
+Erasure Code Limits
+-------------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 75 25
+   :width: 90%
+
+   * - Item
+     - Specification 
+
+   * - Maximum number of servers per cluster
+     - no limit
+
+   * - Minimum number of servers
+     - 1
+
+   * - Minimum number of drives per server when server count is 1
+     - 2
```

Comment:
This should be 1 now right, since *technically* we support SNSD w/ 0-parity erasure coding?

Or we can state that explicitly here with a note that "SNSD Deployments support 1x1 setups in a 0-parity mode that provides no failover tolerance."

Reply:
Fixed.